content/en/docs/reference/scheduling/config.md

```diff
@@ -1,30 +1,57 @@
 ---
-title: Scheduling Profiles
+title: Scheduler Configuration
 content_type: concept
 weight: 20
 ---
 
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
+
+The `KubeSchedulerConfiguration` is a configuration API for `kube-scheduler`
+that can be provided as a file via `--config` command line flag.
+
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
+<!-- body -->
 
-A scheduling Profile allows you to configure the different stages of scheduling
-in the {{< glossary_tooltip text="kube-scheduler" term_id="kube-scheduler" >}}.
-Each stage is exposed in a extension point. Plugins provide scheduling behaviors
-by implementing one or more of these extension points.
+## Minimal Configuration
 
-You can specify scheduling profiles by running `kube-scheduler --config <filename>`,
-using the component config APIs
-([`v1alpha1`](https://pkg.go.dev/k8s.io/kube-scheduler@{{< param "fullversion" >}}/config/v1alpha1?tab=doc#KubeSchedulerConfiguration)
-or [`v1alpha2`](https://pkg.go.dev/k8s.io/kube-scheduler@{{< param "fullversion" >}}/config/v1alpha2?tab=doc#KubeSchedulerConfiguration)).
-The `v1alpha2` API allows you to configure kube-scheduler to run
-[multiple profiles](#multiple-profiles).
+A minimal configuration looks as follows:
 
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1beta1
+kind: KubeSchedulerConfiguration
+clientConnection:
+  kubeconfig: /etc/srv/kubernetes/kube-scheduler/kubeconfig
+```
 
+## Upgrading from `v1alpha2` to `v1beta1` {#beta-changes}
```

Comment:
I would move the “Upgrading from” to the end of this page, just before ` ## {{% heading "whatsnext" %}}`

Reply:
Done